nemo/core/classes/exportable.py

```diff
@@ -153,34 +151,30 @@ def _export(
                         check_trace_input = [input_example]
                     else:
                         check_trace_input = check_trace
-
+                jitted_model = self
```

Comment:
do we really need such an alias for the `self` instance? It seems redundant.

Reply:
This is to possibly use jit.trace() first. Now not working due to Torch bugs.